packages/aws-cdk-lib/aws-opensearchservice/lib/domain.ts

```diff
@@ -281,7 +281,9 @@ export enum TLSSecurityPolicy {
   /** Cipher suite TLS 1.0 */
   TLS_1_0 = 'Policy-Min-TLS-1-0-2019-07',
   /** Cipher suite TLS 1.2 */
-  TLS_1_2 = 'Policy-Min-TLS-1-2-2019-07'
+  TLS_1_2 = 'Policy-Min-TLS-1-2-2019-07',
+  /** Cipher suite TLS 1.2 to 1.3 with perfect forward secrecy (PFS) */
+  TLS_1_2_PFS = 'Policy-Min-TLS-1-2-PFS-2023-10',
```

Comment:
```suggestion
  TLS_1_3 = 'Policy-Min-TLS-1-2-PFS-2023-10',
```
I think it's clearer if we specify `v1.3`.

Reply:
Thanks for your review.

I thought about that too, but do we not have to think about the future appearance of a new option that only allows TLS 1.3 (does not include 1.2)?

Also, there is hesitation in key names that are not in line with the value, what do you think?